src/components/Group.js

```diff
@@ -2,20 +2,21 @@
 import React, { type Node, type ComponentType } from 'react';
 import { css } from 'emotion';
 
-import { spacing } from '../theme';
 import type { CommonProps } from '../types';
 
 type ComponentProps = {
   /** The children to be rendered. */
   children: Node,
   /** Component to wrap the label, recieves headingProps. */
   Heading: ComponentType<any>,
+  /** Props to pass to Heading. */
+  headingProps: any,
```

Comment:
This is new. @akx What's the plan here?

Reply:
It was missing, but referenced in the comment for `Heading` above:

```javascript
/** Component to wrap the label, recieves headingProps. */
```

Reply:
This needs to be updated to pull theme out before passing the props on to the actual HTML input element.